src/components/RadioButton.js/RadioButton.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';;
```

Comment:
Also, you have two `;` here, and to be on the safe side, you're better of naming the directory `RadioButton` instead of `RadioButton.js` just to avoid unnecessary conflicts.

Reply:
Maybe `radioButton` just to follow the camelcase? I'll add these naming conventions somewhere in the doc to make things easier.